Executables/Game/ParserUtils.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Text.RegularExpressions;
+using EnumsNET;
+using QuantumCore.API.Game.World;
+using QuantumCore.Game.World;
+
+namespace QuantumCore.Game;
+
+internal static partial class ParserUtils
+{
+    public static SpawnPoint? GetSpawnFromLine(string line)
+    {
+        if (string.IsNullOrWhiteSpace(line)) return null;
+        var splitted = SplitByWhitespaceRegex().Split(line.Trim());
+        if (string.IsNullOrWhiteSpace(splitted[0]) || splitted[0].StartsWith("//")) return null;
+        return new SpawnPoint
+        {
+            Type = Enums.Parse<ESpawnPointType>(splitted[0].AsSpan()[..1], true, EnumFormat.EnumMemberValue),
```

Comment:
```suggestion
            Type = Enums.Parse<ESpawnPointType>(splitted[0].AsSpan()[0], true, EnumFormat.EnumMemberValue),
```
As discussed in #22

Reply:
Well these do not work as Enum.Parse expects a `string` or `ReadOnlySpan<char>` and `char` is not valid...